cypress/support/commands/sendEmailAndValidatePasscode.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * @jest-environment jsdom
+ */
+
+declare global {
+	// eslint-disable-next-line @typescript-eslint/no-namespace
+	namespace Cypress {
+		interface Chainable {
+			sendEmailAndValidatePasscode: typeof sendEmailAndValidatePasscode;
+		}
+	}
+}
+
+/**
+ * Send an email, receive the passcode and validate it on the sign in page
+ * @param emailAddress The email address you want the passcode to be sent to
+ * @param expectedReturnUrl  The url you are returned to after signing in
+ * @param params URL params added to the sign in url
+ * @param expectedEmailBody  What do you want the email body copy to say
+ * @param additionalTests  Additional tests/checks that you might want to chek for as well as signining in
+ * @returns A chainable cypress command.
+ */
+export const sendEmailAndValidatePasscode = ({
```

Comment:
Moved this out into a shared cypress command so that it can be used by separate github workers now that the test has been split up into multiple files.